substra/sdk/client.py

```diff
@@ -116,19 +106,37 @@ def from_config_file(
         cls,
         profile_name: typing.Optional[str] = None,
```

Comment:
profile is required: `cfg.ConfigManager(config_path).get_profile(None)` raises ProfileNotFoundError
or use the DEFAULT_PROFILE_NAME variable in the _ProfileManager

Reply:
We could have the DEFAULT_PROFILE_NAME as default here. This way `Client.from_config_file()` returns the default profile of the default config files instead of raising an error.